content/infrastructure/install_gds_stack.yml

```diff
@@ -15,13 +17,10 @@ dependencies:
   - mplleaflet
   - networkx
   - palettable
-  - pip
-  - python=2.7
```

Comment:
If this is noise to you I can put them back here. I just like to place the python on top so I can easily see the version together with `pip` to be aware if pip-install will work or not.